server/src/main/java/org/candlepin/controller/OwnerManager.java

```diff
@@ -57,26 +57,54 @@
-    @Inject private OwnerEnvContentAccessCurator ownerEnvContentAccessCurator;
-    @Inject private UeberCertificateCurator uberCertificateCurator;
-    @Inject private OwnerServiceAdapter ownerServiceAdapter;
-
+    private ConsumerCurator consumerCurator;
+    private ActivationKeyCurator activationKeyCurator;
+    private EnvironmentCurator envCurator;
+    private ExporterMetadataCurator exportCurator;
+    private ImportRecordCurator importRecordCurator;
+    private PermissionBlueprintCurator permissionCurator;
+    private OwnerProductCurator ownerProductCurator;
+    private ProductManager productManager;
+    private OwnerContentCurator ownerContentCurator;
+    private ContentManager contentManager;
+    private OwnerCurator ownerCurator;
+    private ContentAccessCertServiceAdapter contentAccessCertService;
+    private ContentAccessCertificateCurator contentAccessCertCurator;
+    private OwnerEnvContentAccessCurator ownerEnvContentAccessCurator;
+    private UeberCertificateCurator uberCertificateCurator;
+    private OwnerServiceAdapter ownerServiceAdapter;
+
+    @Inject
+    public OwnerManager(ConsumerCurator consumerCurator,
+        ActivationKeyCurator activationKeyCurator, EnvironmentCurator envCurator,
+        ExporterMetadataCurator exportCurator, ImportRecordCurator importRecordCurator,
+        PermissionBlueprintCurator permissionCurator, OwnerProductCurator ownerProductCurator,
+        ProductManager productManager, OwnerContentCurator ownerContentCurator,
+        ContentManager contentManager, OwnerCurator ownerCurator,
+        ContentAccessCertServiceAdapter contentAccessCertService,
+        ContentAccessCertificateCurator contentAccessCertCurator,
+        OwnerEnvContentAccessCurator ownerEnvContentAccessCurator,
+        UeberCertificateCurator uberCertificateCurator, OwnerServiceAdapter ownerServiceAdapter) {
+
+        this.consumerCurator = consumerCurator;
+        this.activationKeyCurator = activationKeyCurator;
+        this.envCurator = envCurator;
+        this.exportCurator = exportCurator;
+        this.importRecordCurator = importRecordCurator;
+        this.permissionCurator = permissionCurator;
+        this.ownerProductCurator = ownerProductCurator;
+        this.productManager = productManager;
+        this.ownerContentCurator = ownerContentCurator;
+        this.contentManager = contentManager;
+        this.ownerCurator = ownerCurator;
+        this.contentAccessCertService = contentAccessCertService;
+        this.contentAccessCertCurator = contentAccessCertCurator;
+        this.ownerEnvContentAccessCurator = ownerEnvContentAccessCurator;
+        this.uberCertificateCurator = uberCertificateCurator;
+        this.ownerServiceAdapter = ownerServiceAdapter;
+    }
     @Transactional
-    public void cleanupAndDelete(Owner owner, boolean revokeCerts) {
+    public void cleanupAndDelete(Owner owner, PoolManager poolManager, boolean revokeCerts) {
```

Comment:
Is there another way to do this that isn't completely painful? I'm not a huge fan of rearranging production code to appease testing frameworks.